tests/views_integration/README.md

```diff
@@ -316,10 +316,155 @@ This second test example shows what is, probably, the main feature of data-drive
 * we manipulate data as the full View, in production, would do
 
 ### Snapshot testing
-@todo @juanfra
+
+This type of testing answers the following question:
+
+> Is this View, or View partial, rendering the correct markup (HTML structure, attributes, and data output) given a specific set of template variables?
```

Comment:
@juanfra  @paulmskim  If we use snapshot testing in the following section, then we need to remove the mention of partials from here.
This sections should deal with entire Views only and should be renamed "View snapshot testing".